documentcloud/documents/views.py

```diff
@@ -911,3 +914,18 @@ def post_process(self, request, document_pk=None):
         post_process.delay(document_pk, request.data)
 
         return Response("OK", status=status.HTTP_200_OK)
+
+
+class FreestandingEntityViewSet(viewsets.ModelViewSet):
```

Comment:
I would rename this to `EntityViewSet` and rename the current `EntityViewSet` to `EntityOccurenceViewSet` as that would be more accurate.